Cargo.toml

```diff
@@ -43,7 +43,7 @@ rustc-hash = "2.0.0"
 rrs-lib = "0.1.0"
 tracing = "0.1.40"
 
-womir = { git = "https://github.com/powdr-labs/womir.git", rev = "bda63a1" }
+womir = { git = "https://github.com/powdr-labs/womir.git", rev = "73bf8c7" }
```

Comment:
This is already in main since https://github.com/powdr-labs/womir-openvm/pull/76/files, I don't know why it shows here